tsconfig.json

```diff
@@ -14,7 +14,8 @@
       "@catalog/*": ["src/views/catalog/*"],
       "@kubevirt-utils": ["src/utils"],
       "@kubevirt-utils/*": ["src/utils/*"],
-      "@virtualmachines/*": ["src/views/virtualmachines/*"]
+      "@virtualmachines/*": ["src/views/virtualmachines/*"],
+      "react": ["./node_modules/@types/react"]
```

Comment:
this should be imported automatically

Reply:
Unfortunately, it doesn't work well without this change.